packages/drawer/.storybook/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Is this required?  Storybook config is based on CRA already.

Reply:
We needed for something else that didn't pan out so yeah, don't need it anymore.